docs/README.md

```
@@ -0,0 +1,76 @@
+
+## Prerequisites for Developers
+
+The contents of the `talawa-admin` repo is used to automatically create [the talawa-admin Documentation website](https://docs-admin.talawa.io/). The automation uses [Docusaurus](https://docusaurus.io/docs/), a modern static website generator.
+
+We recommend that you follow these steps before beginning development work in this repository.
+
+### Install the Required NPM Package Manager
+
+For the package installation use `npm`. The steps are simple:
+
+1. Validate the `npm` installation on your local device by running the following command. You should get the `npm` version.
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)